scripts/deploy_infrastructure.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -ex
+
+ENV=$1
+TERRAFORM_STATE_BUCKET="${TERRAFORM_STATE_BUCKET:-census-eq-terraform-tfstate}"
+
+terraform init --upgrade --backend-config prefix=${ENV} --backend-config bucket=${TERRAFORM_STATE_BUCKET}
+
+if [ ! -z $EXISTING_PROJECT_ID ]; then
+    echo "Using existing project_id: $EXISTING_PROJECT_ID"
+
+    if terraform state list | grep -q "google_project.project"; then
+        echo "State contains a google project. Not importing"
+    else
+        echo "State does not contain a google project. Importing $EXISTING_PROJECT_ID"
+        terraform import google_project.project $EXISTING_PROJECT_ID
+    fi
+fi
+
+# Roll out infrastructure
+terraform apply -auto-approve -var 'env=${ENV}'
```

Comment:
This is failing with the error:

```
Error: google_project.project: "${ENV}" name must be 4 to 30 characters with lowercase and uppercase letters, numbers, hyphen, single-quote, double-quote, space, and exclamation point.
```

`${ENV}` isn't being replaced in the string as the string is single rather than double quoted in the script.